plugin/build.sbt

```diff
@@ -6,7 +6,7 @@ sbtPlugin := true
 
 publishMavenStyle := false
 
-bintrayOrganization in bintray := Some("tpolecat")
+//bintrayOrganization in bintray := Some("tpolecat")
```

Comment:
Ah crap. No, I had to disable that to get things to work locally (don't have bintray credentials, couldn't seem to make sbt-bintray happy). That's part of the local mixup I had, I was pretty sure I'd nuked them all. Terribly sorry about this :(